src/app/static/src/tests/components/reportLog/runningReportsList.test.ts

```diff
@@ -1,63 +1,61 @@
+    {name: "report1", date: new Date(2021, 3, 21, 10, 1).toISOString(), key: "report1Key"}
+];
+
+function getWrapper(key = "") {
     return mount(RunningReportsList, {
         propsData: {
             initialSelectedKey: key,
-            reports: [
-                {name: "report2", date: new Date(2021, 3, 21, 9, 10).toISOString(), key: "report2Key"},
-                {name: "report1", date: new Date(2021, 3, 21, 10, 1).toISOString(), key: "report1Key"}
-            ]
+            reports
         }
     });
 }
 
 describe("runningReportsList", () => {
 
-    it("renders typeahead correctly and fires event on selection", (done) => {
+    it("renders typeahead correctly and fires event on selection", async () => {
         const wrapper = getWrapper();
 
-        const reportSuggestions = wrapper.findAll("a div.sr-only");
+        (wrapper.findComponent(VueSelect).vm.$refs.search as any).focus()
+        await Vue.nextTick();
+
+        const reportSuggestions = wrapper.findAll("li");
         expect(reportSuggestions.length).toBe(2);
-        expect(reportSuggestions.at(0).text()).toBe("report1");
-        expect(reportSuggestions.at(1).text()).toBe("report2");
-        reportSuggestions.at(1).trigger("click");
+        expect(reportSuggestions.at(0).text()).toBe("report1 Run started: Wed Apr 21 2021, 10:01");
+        expect(reportSuggestions.at(1).text()).toBe("report2 Run started: Wed Apr 21 2021, 09:10");
+
+        (wrapper.findComponent(VueSelect).vm as any).select(reports[0]);
+        await Vue.nextTick();
         expect(wrapper.emitted()["update:key"].length).toBe(1);
         expect(wrapper.emitted()["update:key"][0]).toEqual(["report2Key"]);
-
-        done();
     });
 
-    it("typeahead filters list correctly", async (done) => {
+    it("typeahead filters list correctly", async () => {
         const wrapper = getWrapper();
 
-        wrapper.find("input").setValue("rt2");
-
+        (wrapper.findComponent(VueSelect).vm.$refs.search as any).focus();
         await Vue.nextTick();
 
-        const reportSuggestions = wrapper.findAll("a div.sr-only");
+        (wrapper.findComponent(VueSelect).vm as any).search = "rt2";
+        await Vue.nextTick();
+        const reportSuggestions = wrapper.findAll("li");
         expect(reportSuggestions.length).toBe(1);
-        expect(reportSuggestions.at(0).text()).toBe("report2");
-
-        const reportDates = wrapper.findAll("a span.text-muted");
-        expect(reportDates.length).toBe(1);
-        expect(reportDates.at(0).text()).toBe("Run started: Wed Apr 21 2021, 09:10");
-
-        done();
+        expect(reportSuggestions.at(0).text()).toContain("report2");
     });
 
-    it("typeahead comes with correct report name preselected", async (done) => {
-        const wrapper = getWrapper("report1Key");
+    it("typeahead comes with correct report name preselected", async () => {
+        const wrapper = getWrapper("report2Key");
```

Comment:
Switching this to report 2, purely because it's not the first in the list, so slightly more rigorous test(?)